x/admin.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022-2025 Searis AS
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clarifyx
+
+import (
+	"github.com/clarify/clarify-go/internal/request"
+	"github.com/clarify/clarify-go/views"
+)
+
+type SignalAdminGroup struct {
+	AdminNamespace
+}
```

Comment:
Should we let the group have a plural name to match the path? And should we order Admin first, same as in the path?

`AdminSignalsGroup`

Reply:
Could maybe drop Group `AdminSignals`...

Reply:
Related question, do we want to move Connect / Disconnect to the Signals group (in the SDK)?

The API paths could be updated later...

Reply:
Agree! I have updated the commit to use `AdminSignals`.